constraints.txt

```diff
@@ -0,0 +1,2 @@
+# 2.32.2 can lead to invalid account error (see #2474)
+requests!=2.32.2
```

Comment:
@ElePT the issue explicitly mentioned excluding the offending version via this mechanism, but is this file actually being used by the project (or other projects consuming it) currently?

As far as I can tell, since around #2053 the move to `pyproject.toml` was completed, and along with it the references to `pip install [...] -c constraints.txt`.